### DISCUSSION TITLE
Installation instructions for OS X not compatible with Apple Silicon

### DISCUSSION THREAD

**Discussion by njern** [thinkst/opencanary#274] [Feature Requests]
Hello,

Recently built this for my M1 Macbook and realised the existing [instructions](https://github.com/thinkst/opencanary#installation-os-x) only work for x86-based OS X computers.

I suggest updating the documentation. 

Trying this out myself, when using Homebrew, it is easy to get it working by replacing the following line:

```
env ARCHFLAGS="-arch x86_64" LDFLAGS="-L/usr/local/opt/openssl/lib" CFLAGS="-I/usr/local/opt/openssl/include" pip install cryptography
```

With:

```
env ARCHFLAGS="-arch arm64" LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib" CFLAGS="-I/opt/homebrew/opt/openssl@1.1/include" pip install cryptography
```

**Comment by jayjb** [comment]
Hi @njern,

Thanks! Ill give it a shot. If it works, maybe you could submit a PR?

**Comment by jayjb** [comment]
These instructions have been added
